MovieNight/ActorViewController.swift

```diff
@@ -97,8 +98,16 @@ class ActorViewController: UIViewController, UITableViewDataSource, UITableViewD
     
     // Called when the user taps 'Next' button
     @IBAction func saveActorsSelectedInDisk(_ sender: Any) {
-        repository.save(dictionary: actorsSelected, forKey: UserKeys.FoxUserActors.rawValue)
+        switch user {
```

Comment:
Here you'd better do the switch to decide the key and then call `repository.save` only once.